src/transformer/create-mock-list.ts

```diff
@@ -0,0 +1 @@
+export declare function createMockList<T extends object>(quantity?: number): T[];
```

Comment:
I would prefer the quantity to be mandatory, is there a reason behind making it optional?

Reply:
I was thinking when you want just to create an array of one... I am not sure is useful. We can remove it for now.

Reply:
I was thinking that it was creating an empty array, it's not clear what is going to do without input